test/cassandrax/queryable_test.exs

```diff
@@ -1,4 +1,6 @@
 defmodule Cassandrax.QueryableTest do
+  @moduledoc """
+  """
```

Comment:
I don't think we need this, but if you're trying to get rid of warnings here, just do `@moduledoc false`